packages/webview_flutter/lib/src/tizen_webview.dart

```diff
@@ -139,28 +137,29 @@ class TizenWebView {
   /// Returns the title of the currently loaded page.
   Future<String?> getTitle() => _invokeChannelMethod<String>('getTitle');
 
-  /// Set the scrolled position of this view.
+  /// Sets the scrolled position of this view.
   Future<void> scrollTo(int x, int y) =>
       _invokeChannelMethod<void>('scrollTo', <String, int>{
         'x': x,
         'y': y,
       });
 
-  /// Move the scrolled position of this view.
+  /// Moves the scrolled position of this view.
   Future<void> scrollBy(int x, int y) =>
       _invokeChannelMethod<void>('scrollBy', <String, int>{
         'x': x,
         'y': y,
       });
 
-  /// Return the set scroll position of this view.
+  /// Returns the scroll position of this view set by [scrollTo].
   Future<Offset> getScrollPosition() async {
-    final dynamic position =
-        await _invokeChannelMethod<dynamic>('getScrollPosition');
+    final Map<Object?, Object?>? position =
+        await _invokeChannelMethod<Map<Object?, Object?>?>('getScrollPosition');
```

Comment:
A better practice is to use `MethodChannel.invokeMapMethod`, but for now we keep using `_invokeChannelMethod`.